resources/lang/km/passwords.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Password Reset Language Lines
+    |--------------------------------------------------------------------------
+    |
+    | The following language lines are the default lines which match reasons
+    | that are given by the password broker for a password update attempt
+    | has failed, such as for an invalid token or invalid new password.
+    |
+    */
+
+    'reset' => 'ពាក្យសម្ងាត់បានផ្លាស់ប្តូរជាថ្មី។',
+    'sent' => 'ពាក្យសម្ងាត់ដែលបានរំលឹកបានផ្ញើរួចរាល់!',
+    'throttled' => 'សូមរង់ចាំមុនពេលសាកល្បងម្តងទៀត។',
+    'token' => 'ពាក្យសម្ងាត់ដែលកំណត់ឡើងវិញគឺមិនត្រឹមត្រូវទេ។',
+    'user' => "យើងមិនអាចរកឃើញអ្នកប្រើដែលមានអាសយដ្ឋានអ៊ីមែលមួយនេះទេ។",
```

Comment:
@HELMAB you might wanna look into this particular line, as StyleCI is flagging this as the check fail.

Reply:
Thank you @coderkoala I have updated it